localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -397,6 +397,15 @@ void EKFLocalizer::initEKF()
   ekf_.init(X, P, params_.extend_state_step);
 }
 
+void warnIfPoseDelayTimeLessThanZero(const Warning & warning, const double delay_time)
```

Comment:
(Don't necessarily have to fix this, just a question)
Putting a non-member function between two member functions may be confusing. What do you think?

Reply:
This function is used quite locally, at least within this file so I think there's no problem